app/frontend/src/components/Stats.vue

```diff
@@ -1,79 +1,113 @@
+        </template>
+
+        <template v-slot:item.test="{item}">
+          <v-icon v-if="item.test === 1" color="green">check</v-icon>
+        </template>
+
+        <template v-slot:item.prod="{item}">
+          <v-icon v-if="item.prod === 1" color="green">check</v-icon>
+        </template>
+      </v-data-table>
     </template>
-  </v-container>
+  </v-card>
 </template>
 
 
 <script>
-import statsService from '@/services/statsService';
+import keycloakService from '@/services/keycloakService';
 
 export default {
   name: 'Stats',
   data() {
     return {
+      // vuetify data table
+      search: '',
       headers: [
-        { text: 'Service Client', align: 'start', value: 'name' },
-        { text: 'Created', value: 'created' },
+        { text: 'Name', align: 'start', value: 'name' },
         { text: 'DEV', value: 'dev' },
         { text: 'TEST', value: 'test' },
         { text: 'PROD', value: 'prod' }
       ],
-      serviceClients: this.getData()
-      /*
-      [
-        {
-          name: 'ABC_ONE',
-          created: 159,
-          dev: 1,
-          test: 0,
-          prod: 0
-        },
-        {
-          name: 'DEF_TWO',
-          created: 237,
-          dev: 1,
-          test: 1,
-          prod: 0
-        },
-        {
-          name: 'HKI_THREE',
-          created: 262,
-          dev: 1,
-          test: 0,
-          prod: 0
-        },
-        {
-          name: 'BED_FOUR',
-          created: 305,
-          dev: 1,
-          test: 1,
-          prod: 1
-        }
-      ]
-      */
+      serviceClients: [],
+      loading: true
     };
   },
-
+  watch: {
+    // hide data table progress bar when serviceClients have been returned from backend
+    serviceClients() {
+      this.loading = false;
+    }
+  },
   methods: {
     // get table data from frontend service layer
     getData() {
-      statsService
-        .getServiceClients()
-        .then(response => {
-          if (response) {
-            return response;
+      keycloakService.getServiceClients().then(response => {
+        if (response) {
+
+          // reformat serviceClients data into array of objects that will work for the data table
+          var output = [];
+          var tmpArray = [];
+          response.data.forEach(sc => {
+            // if object doesnt exist in tmpArray
+            if (tmpArray[sc.clientId] == undefined) {
+              //create empty object
+              tmpArray[sc.clientId] = {};
+            }
+            // update object
+            tmpArray[sc.clientId].name = sc.clientId;
+            if (sc.realm === 'dev') tmpArray[sc.clientId].dev = 1;
+            if (sc.realm === 'test') tmpArray[sc.clientId].test = 1;
+            if (sc.realm === 'prod') tmpArray[sc.clientId].prod = 1;
+          });
+          //convert back to numeric array
+          for (var items in tmpArray) {
+            output.push(tmpArray[items]);
           }
```

Comment:
Not sure what this loop is really trying to achieve. This looks like you're effectively doing
`output = tmpArray` as you don't use output anywhere else and getting all elements transferred over.